portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -890,6 +901,19 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 		    d.cosmic_count = cosmic_count;
 		    ret = ret || ((typeof cosmic_count !== "undefined") && (cosmic_count >= known_mutation_settings.cosmic_count_thresh));
 		}
+		if (known_mutation_settings.recognize_binary_filter) {
+		    if (d.binary_filter == "PUTATIVE_DRIVER") {
+			ret = true; //Set the element to be a driver
+		    }
+		}
+		if (known_mutation_settings.recognize_multi_filter) {
+		    for (var value in getMultiValuesMap()) {
```

Comment:
to do this safely you need to add a `hasOwnProperty` check, or you can use a lo-dash function to iterate